Sources/RInAppMessaging/Views/Presenters/FullViewPresenter.swift

```diff
@@ -114,6 +115,18 @@ internal class FullViewPresenter: BaseViewPresenter, FullViewPresenterType, Erro
 
         view?.dismiss()
     }
+    
+    func didClickCampaignImage() {
+        guard !campaign.isPushPrimer else { return }
+        guard let clickImageData = campaign.data.customJson?.clickableImage,
+              let uriToOpen = URL(string: clickImageData.url ?? "") else {
+            return
+        }
+        logImpression(type: .clickContent)
+        sendImpressions()
+        UIApplication.shared.open(uriToOpen)
```

Comment:
what will happen if uriToOpen is empty string ?

Reply:
Expected behaviour is to do nothing on tap. It will behave as a normal campaign.
This is covered in the test cases .